.circleci/config.yml

```diff
@@ -2,7 +2,6 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/openjdk:9.0.4-jdk # java 9
```

Comment:
I guess it is okay to remove this. anything compiled, tested and published with jdk8 should be usable with jdk9 as well.